.travis.yml

```diff
@@ -193,17 +195,20 @@ matrix:
     - env:
         - DESC="spell checker"
         - CMD="./.ci/test-spelling-unknown-words.sh"
+        - SKIP_CI="false"
 
 script:
   - SKIP_FILES=".github|codeship-*|buddy.yml|appveyor.yml|circleci|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intellij-idea-inspections.xml"
-  - SKIP_CI=$(if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then echo false; else echo true; fi;)
+  - SKIP_CI=$(if [[ $SKIP_CI == 'false' || $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then echo false; else echo true; fi;)
```

Comment:
it was already hard to read , but I did not have time improve it.
one we hitting this ground
lets make it more human readable:
```
if [[ $SKIP_CI != 'false' ]] then 
  if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then 
    SKIP_CI=false;
  else
    SKIP_CI=true;
  fi;
fi

```

Reply:
Done.